### PR TITLE
Add setInitialRoute on FlutterView.

### PR DESCRIPTION
**Pull request by szakarias**

### CODE REVIEW

**Comment by Hixie** [comment]
cc @abarth who is more familiar with this code

**Comment by Hixie** [comment]
Also cc @jason-simmons who was in this code recently.

File: lib/ui/window/window.cc [hunk 1]
```diff
@@ -39,6 +39,11 @@ Dart_Handle ToByteData(const std::vector<uint8_t>& buffer) {
   return data_handle;
 }
 
+void DefaultRouteName(Dart_NativeArguments args) {
+    std::string routeName = UIDartState::Current()->window()->client()->DefaultRouteName();
+    Dart_SetReturnValue(args, Dart_NewStringFromCString(routeName.c_str()));
```
Comment (abarth) [inline lib/ui/window/window.cc]: You can just call: `Dart_SetReturnValue(args, StdStringToDart(routeName));`

File: shell/common/engine.cc [hunk 2]
```diff
@@ -450,6 +448,13 @@ void Engine::StartAnimatorIfPossible() {
     animator_->Start();
 }
 
+std::string Engine::DefaultRouteName() {
+  if (!initial_route_.empty()) {
+    return initial_route_;
+  }
+  return std::string("/");
```
Comment (abarth) [inline shell/common/engine.cc]: No need for the explicit call to `std::string`.  The compiler will add it for you.

**Comment by abarth** [comment]
LGTM

**Pull request merged**